content/code-challenges/dappworld.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "DApp World"
+description: "DApp World is a blockchain upskilling platform where developers learn by doing!"
+authors: ["@DApp World"]
+tags: ["Smart Contracts", "Solidity", "Challenges"]

```

Comment:
Can you remove 'challenges' from the tags? I'd like to stick with tags that are available at https://www.useweb3.xyz/tags

Thank you!

Reply:
Hey, Thank you.
I have replaced the "Challenges" tag with "Web3" from the available tags.